pkg/services/authn/authn.go

```diff
@@ -94,6 +94,9 @@ type Service interface {
 
 	// RegisterClient will register a new authn.Client that can be used for authentication
 	RegisterClient(c Client)
+
+	// GetClient returns a client by name
+	GetClient(name string) Client
```

Comment:
As I said in the other pr, I think this should be IsClientEnabled and just proxy the call to client. Not sure we should expose this externally